src/pages/settings/Report/ReportSettingsPage.tsx

```diff
@@ -88,23 +59,23 @@ function ReportSettingsPage(props) {
             <FullPageNotFoundView shouldShow={shouldDisableSettings}>
                 <HeaderWithBackButton
                     title={translate('common.settings')}
-                    onBackButtonPress={() => Navigation.goBack(ROUTES.REPORT_WITH_ID_DETAILS.getRoute(report.reportID))}
+                    onBackButtonPress={() => Navigation.goBack(ROUTES.REPORT_WITH_ID_DETAILS.getRoute(report?.reportID ?? ''))}
```

Comment:
What about extracting `report?.reportID ?? ''` to a new variable and use in every occurence we have in this file?